proposals/extensions.md

```diff
@@ -882,11 +883,9 @@ and when successful this process yields an extension type we can use
 
 An extension type `X` is compatible with given type `U` if:
 - `X` is non-generic and its underlying type is `U`, a base type of `U` or an implemented interface of `U`
-- a possible type substitution on the type parameters of `X` yields underlying type `U`, 
+- we collect all the possible type substitutions on the type parameters of `X` yields underlying type `U`, 
```

Comment:
>we collect all the possible type substitutions on the type parameters of `X` yields underlying type `U`, [](http://example.com/codeflow?start=2&length=104)

The wording doesn't quite work in the list of "An extension type `X` is compatible with given type `U` if"  conditions. This item describes some form of action, not a condition.